modules/steam/hm.nix

```
@@ -0,0 +1,60 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.stylix.targets.steam;
+
+  adwaitaCustomCss = config.lib.stylix.colors {
+    template = ./custom.mustache;
+    extension = "css";
+  };
+
+  adwaitaTheme = pkgs.stdenv.mkDerivation (_self: {
```

Remove this `_self:` clause:

```suggestion
  adwaitaTheme = pkgs.stdenv.mkDerivation {
```